runtime/cudaq/platform/quantum_platform.h

```diff
@@ -133,7 +133,7 @@ class quantum_platform {
                     void *args, std::uint64_t voidStarSize,
                     std::uint64_t resultOffset);
 
-  /// List all available platforms, which correspond to .qplt files in the
+  /// List all available platforms, which correspond to `.qplt` files in the
```

Comment:
The qplt part can be removed, old comment from way back when.

Reply:
Fixed.